Colorful/Colorful.js

```diff
@@ -0,0 +1,54 @@
+tau.mashups
+    .addDependency('tp3/mashups/topmenu')
+    .addMashup(function(topMenu) {
+        'use strict';
+        var initColorfulTheme = function() {
+            var style = document.createElement("style");
+            style.setAttribute('id', 'qs-color-style');
```

Comment:
Suggest to change id here in case somebody want to install original colorful mashup / use it as base for its own one.